.circleci/config.yml

```diff
@@ -58,6 +58,10 @@ commands:
 
             [[ "<< parameters.event >>" == "failure" ]] && message=" ❌ [${workflow_name}] Job ${job} failed on **${CIRCLE_BRANCH}**. Please see [build ${CIRCLE_BUILD_NUM}](${CIRCLE_BUILD_URL}) for details."
             [[ "<< parameters.event >>" == "success" ]] && message=" ✅ [${workflow_name}] Job ${job} succeeded on **${CIRCLE_BRANCH}**. Please see [build ${CIRCLE_BUILD_NUM}](${CIRCLE_BUILD_URL}) for details."
+            [[ "<< parameters.event >>" == "release" ]] && message=" 📦 Release binaries for version **${CIRCLE_TAG}** are ready and attached as artifacts to [build ${CIRCLE_BUILD_NUM}](${CIRCLE_BUILD_URL}). **Please make sure the whole workflow succeeded before using them.**"
+
+            # The release notification only makes sense on tagged commits
+            [[ "<< parameters.event >>" == "release" ]] && { [[ $CIRCLE_TAG != "" ]] || { echo "Not a tagged commit - notification skipped."; exit 0; } }
```

Comment:
I am a bit confused, the message in line 64 basically locally echos that it will skip the notification, while line 61 actually does post the notification.. wouldn't line 61 need that extra condition? a local echo that we won't post a notification on its own doesn't seem very useful to me. But I am probably missing some logic here..?

Reply:
Line 61 just sets the value of the `message` variable. The code that actually posts it is the `curl` command below. If line 64 exits, the message won't actually be used.

I could move line 61 lower or put 61 and 64 under a single `if` but I thought it was nicer to have these messages side by side for easy comparison.

Reply:
Done.